tests/puppeteer.js

```diff
@@ -59,7 +59,7 @@ chai.should();
         console.log("at 1, resultData is ", resultData);
         resultData.should.include("A person record was added");
         const { index } = JSON.parse(resultData);
-        this.lastIndex = index;
+        this.Index = index;
       });
       it("should not create a person record without an age", async function () {
```

Comment:
The chai tests look good but it looks like all the puppeteer tests are missing.

Reply:
Hey sorry for not answering before, I was working in my project and to be honest I kind of give up in this assignment. I tried to understand the puppeteer test but I didn't get to far.... srry